packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.tsx

```diff
@@ -57,11 +59,11 @@ const ModalStepWrapper: FC<PropsWithChildren<TModalStepWrapperProps>> = ({
             )}
             <div className='wallets-modal-step-wrapper__body'>
                 {children}
-                {!shouldFixedFooter && hasRenderFooter && (
+                {!shouldHideFooter && !shouldFixedFooter && hasRenderFooter && (
```

Comment:
for future PRs - generally positive logic is easier to understand and our brains cope with it better

one of examples: https://learn.microsoft.com/en-us/dynamicsax-2012/developer/x-standards-if-else-and-switch-statements?redirectedfrom=MSDN#ifelse